nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/pom.xml

```diff
@@ -68,6 +68,11 @@
             <artifactId>nifi-record</artifactId>
             <version>1.11.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
```

Comment:
Does it require something in the L&N files?

Reply:
I don't believe so, it is ASL 2.0 licensed with no NOTICE in the project.